server/backend/database/memory/database.go

```diff
@@ -224,8 +224,8 @@ func (d *DB) CreateProjectInfo(
 	return info, nil
 }
 
-// listProjectInfos returns all project infos rotationally.
-func (d *DB) listProjectInfos(
+// rotateListProjectInfosGreaterThan returns all project infos rotationally.
+func (d *DB) rotateListProjectInfosGreaterThan(
```

Comment:
How about naming this function to just `listProjectInfosGreaterThan()` or other names? For me, the word `rotate` seems ambiguous.

Reply:
Sorry, It is my mistake.

I changed the function name `rotateListProjectInfosGreaterThan` to `nextCyclingProjectInfos`. How about this?

Reply:
Looks good to me 😄